docs/users-guide/03-basic-exploration.md

```diff
@@ -69,6 +69,26 @@ Searches take into account items’ titles, descriptions, and other metadata —
 
 ![Search results](./images/basic-exploration/search-results.png)
 
+## Bookmarks
+
+**Bookmarks** are a way to quickly get back to stuff you visit frequently (or have been working on recently). Bookmarked items show up in the main, left sidebar above [collections][collections].
+
+To bookmark an item, look for the ribbon icon. If you don't immediately see the ribbon, open up the settings for that item. You can bookmark:
+
+- Questions (ribbon's in the editing sidebar)
+- Models (ribbon's in the editing sidebar)
+- Dashboards (ribbon's in the **...** menu)
+- Collections (ribbon's in the upper right of the collection's page)
+
+When viewing a collection, you can also click on the **...** next to an item to add or remove a bookmark from the item.
+
+Some things to remember with bookmarks:
+
+- Bookmarks are personal; other people can't see your bookmarks. If you want to highlight something for everyone, you'll want to put it in an official collection and/or pin the item in the collection (see [collections][collections]).
+- If you end up bookmarking a lot of items, you can collapse the bookmarks section in the sidebar (or remove the bookmarks that are just getting in your way).
+- You can drag bookmarks to reorder them in the sidebar.
```

Comment:
Drag-to-reorder may not make it, depending on when the release is actually going to be. The FE is done, but not the BE.